platform/android/display_server_android.cpp

```diff
@@ -161,6 +161,15 @@ int DisplayServerAndroid::screen_get_dpi(int p_screen) const {
 	return godot_io_java->get_screen_dpi();
 }
 
+float DisplayServerAndroid::screen_get_refresh_rate(int p_screen) const {
+	GodotIOJavaWrapper *godot_io_java = OS_Android::get_singleton()->get_godot_io_java();
+	if (!godot_io_java) {
+		return 60.0; //Couldn't get refresh rate, default to 60;
```

Comment:
Should this value be a `DisplayServer` constant? That would make things easier if we decide to follow another approach (e.g: use a negative value when the refresh rate is inaccessible).

Reply:
I agree it would be better to use a constant, and explain why the default is 60 in a comment. (It's the most common refresh rate as of 2022.)

Reply:
Good Idea, will implement today.

Reply:
Perhaps there should also be a ERR_PRINT or WARN_PRINT here to let the user know that there was an error.

Reply:
Implemented, waiting on checks to pass.